migrator/migrations/m_213_to_m_214_populate_deployment_containers_imageidv2/migration_impl.go

```diff
@@ -0,0 +1,60 @@
+package m213tom214
+
+import (
+	"github.com/cloudflare/cfssl/log"
+	"github.com/stackrox/rox/migrator/migrations/m_213_to_m_214_populate_deployment_containers_imageidv2/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/uuid"
+)
+
+func migrate(database *types.Databases) error {
+	// Use databases.DBCtx to take advantage of the transaction wrapping present in the migration initiator
+	pgutils.CreateTableFromModel(database.DBCtx, database.GormDB, schema.CreateTableDeploymentsStmt)
+
+	db := database.PostgresDB
+
+	getStmt := `SELECT image_name_fullname, image_id FROM deployments_containers WHERE image_id is not null AND image_id != '' AND image_name_fullname is not null AND image_name_fullname != ''`
```

Comment:
Add `image_idv2 is null` to the query too. That way if someone rolled back and upgraded to this version again, the migration can skip already updated containers.